MyOnlineStore/ruleset.xml

```diff
@@ -168,12 +167,12 @@
     <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
         <properties>
             <property name="newlinesCountBetweenOpenTagAndDeclare" value="1"/>
-            <property name="newlinesCountAfterDeclare" value="1"/>
+            <property name="newlinesCountAfterDeclare" value="2"/>
```

Comment:
Why this extra empty line requirement?

Reply:
Two newlines = one empty line. It was wrong before, but not used yet due to the older version.